src/wizards/audience/views/campaigns/segments/single-segment.js

```diff
@@ -396,4 +397,39 @@ addFilter(
 	}
 );
 
+/**
+ * Adds a custom input for the devices criteria so more than one device can be selected.
+ */
+addFilter(
+	'newspack.criteria.input',
+	'newspack.devices',
+	function (element, criteria, value, update) {
+		if (criteria.id === 'devices') {
+			const selectedDevices = Array.isArray(value) ? value : [];
+
+			return (
+				<div className="newspack-device-segments">
+				<VStack spacing={4} alignment="topLeft">
```

Comment:
A `Grid columns={ 1 } rowGap={ 16 }` would work here, without the need for experimental features ;)

Reply:
Sorry didn't know about this one. I meant [_our_ `Grid`](https://github.com/Automattic/newspack-plugin/blob/1008d6ef81bfda3caf78235ca732985a977095aa/src/components/src/grid/index.tsx#L15-L33)

Reply:
TIL! 🙈